master/internal/experiment.go

```diff
@@ -548,21 +548,25 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 		}
 
 	case *apiv1.KillExperimentRequest:
-		switch {
-		case model.StoppingStates[e.State] || model.TerminalStates[e.State]:
```

Comment:
kinda makes sense to me to just change this to
`case e.State == model.StoppingKilledState || model.TerminalStates[e.State]`,
since it could be useful to kill experiments in other stopping states as well.

Reply:
I think that makes more sense

One difference is trying to kill an experiment in ```StoppingErrorState``` will throw an error upon transition since it doesn't have ```StoppingKilledState``` in valid transitions while before it would silently do nothing. That does seem like more ideal behavior to me.